test/integration/ordinary/itest_slc.py

```diff
@@ -187,6 +187,9 @@ def test_deploy_cert_fails(
 def test_deploy(sample_slc: ScriptLanguageContainer, setup_itde_module):
     sample_slc.deploy()
     assert sample_slc.activation_key == expected_activation_key(sample_slc)
+    assert sample_slc.generate_activation_key(False) == sample_slc.secrets.get(
```

Comment:
This is a bit dangerous, because it depends on the execution order, I would first assign both sides to variables